### PR TITLE
Removing the Site Analytics tag

### PR DESCRIPTION
**Pull request by CannonLock**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ baseurl: ''
 exclude: [vendor, deploy_key.enc, TEMPLATE.shtml, archive_html, dev_website]
 
 # Google analytics
-google_analytics: G-5FBPVMZYDF
+google_analytics:
```
Comment (jasoncpatton) [inline _config.yml:5]: Can you explain what happens with the included analytics jscript if the ID is missing? Does it disable it, or do you know if it generates any errors?

Comment (CannonLock) [reply _config.yml:5]: @jasoncpatton In the head there is a check for the presence of the tag, and if it is not present then it skips over the include of the analytic related code.

Comment (CannonLock) [reply _config.yml:5]: https://github.com/CHTC/chtc-website-source/blob/master/_layouts/default.html#:~:text=%7B%25%20if%20site.google_analytics,%7B%25%20endif%20%25%7D

Comment (CannonLock) [reply _config.yml:5]: https://github.com/CHTC/chtc-website-source/blob/596c041e0b0ad21e9c785156ff38f961ca1154a7/_layouts/default.html#L21-L23

**Review by jasoncpatton — APPROVED** [review]

**Pull request merged**